types/go.mod

```diff
@@ -1,9 +0,0 @@
-module github.com/streamingfast/firehose-acme/types
```

Comment:
why is this no longer needed?

Reply:
A bit of a complicated story here, but several reasons:
1. The outter `go`mod` was reffering a commit hash from `filestream/types` repo which I couldn't "manipulate" - main reason
2. There is no need to keep it, there is no main func, only causes overhead
3. It is also deleted in other integration repos
4. I will remove anyway `types` in the future to use proto structs from `elrond-go-core`

Reply:
Refactored it so that we keep the same structure